app/components/ui/tale-tab-metadata/template.hbs

```diff
@@ -7,9 +7,37 @@
     </div>
 
     <div class="inline fields ui grid">
-        <label class="two wide column right aligned">Authors</label>
+      <label class="two wide column right aligned">Authors</label>
+    </div>
+    {{#each taleAuthors as |author|}}
+    <div class="inline fields ui grid">
+      <label class="two wide column right aligned"></label>
+      <div class="field thirteen wide column" id=author.id>
+        <div class="ui form">
+          <div class="inline fields">
+            <label>First Name:</label>
+              <div class="field">
+                {{input type="text" name='first-name' value=author.firstName placeholder="Sur Name" }}
+              </div>
+            <label>Last Name:</label>
+            <div class="field">
+              {{input type="text" name='last-name' value=author.lastName placeholder="Family Name" }}
+            </div>
+          <label>ORCID:</label>
+          <div class="field">
+            {{input type="url" name='orcid-id' placeholder="https://orcid.org/0000-0002-1825-0097"  value=author.orcid }}
+          </div>
+          <i class="fa fa-window-close" aria-hidden="true" style="color: black" {{action 'removeAuthor' author.id}}></i>
+        </div>
+      </div>
+    </div>
+  </div>
+  {{/each}}
+      
+    <div class="inline fields ui grid">
+        <label class="two wide column right aligned"></label>
         <div class="field thirteen wide column">
-            {{input value=model.tale.authors readonly=cannotEditTale}}
+            <label></label> <i class="fas fa-plus" style="color: rgb(44,119,143); margin-left: 10%;"{{action 'addNewAuthor'}}>  Add another author</i>
```

Comment:
`Add another author` here needs to come after the `</i>` - this is why the font is different in this part of the view

When the text is within the icon itself, font-awesome's `font-family` is applied to the text, which breaks consistency with the styling else where in the UI. Placing the text after the icon should render the icon (in `font-awesome` font) and then render the text in our standard font.

Reply:
Nice catch with the styling; I'll try to keep an eye out for situations where semantic ui overrides defaults. I changed the color of the icon and text to match the dark blue bar on the instance inside the instance menu.
<img width="1247" alt="Screen Shot 2019-03-30 at 2 20 49 PM" src="https://user-images.githubusercontent.com/9289652/55282032-6bc64100-52f9-11e9-8ff5-bbd2f2646df6.png">